gulpfile.js

```diff
@@ -60,6 +60,14 @@ gulp.task('bower', function () {
     .pipe(gulp.dest('public/lib'));
 });
 
+// Webpack task
+gulp.task('webpack', function (done) {
```

Comment:
We also do not need the Webpack gulp task. It has been removed from the codebase.